package.json

```diff
@@ -28,12 +28,13 @@
       "!node_modules/realm/src${/*}",
       "!node_modules/realm/vendor${/*}"
     ],
+    "afterSign": "./scripts/afterSign",
     "mac": {
-      "hardenedRuntime": false,
+      "hardenedRuntime": true,
+      "gatekeeperAssess": false,
```

Comment:
Any reason this is `false`? Also - it looks to me as if this is `false` by default? https://www.electron.build/configuration/mac

Reply:
Yes it's now `false` by default, but if true, `electron-builder` will test prematurely. (it could be omitted if preferred?)